rsts/deployment/plugin_setup/k8s/index.rst

```diff
@@ -1,57 +1,414 @@
+                     - spark.blacklist.enabled: "true"
+                     - spark.blacklist.timeout: "5m"
+                     - spark.task.maxfailures: "8"
+           configmap:
+             enabled_plugins:
+               # -- Tasks specific configuration [structure](https://pkg.go.dev/github.com/flyteorg/flytepropeller/pkg/controller/nodes/task/config#GetConfig)
+               tasks:
+                 # -- Plugins configuration, [structure](https://pkg.go.dev/github.com/flyteorg/flytepropeller/pkg/controller/nodes/task/config#TaskPluginConfig)
+                 task-plugins:
+                   # -- [Enabled Plugins](https://pkg.go.dev/github.com/flyteorg/flyteplugins/go/tasks/config#Config). Enable sagemaker*, athena if you install the backend
+                   # plugins
+                   enabled-plugins:
+                     - container
+                     - sidecar
+                     - k8s-array
+                     - spark
+                   default-for-task-types:
+                     container: container
+                     sidecar: sidecar
+                     container_array: k8s-array
+                     spark: spark
+
+5. Upgrade the Flyte Helm release.
+
+   .. tabbed:: Sandbox
+
+        .. code-block:: bash
+
+           helm upgrade -n flyte -f values-override.yaml flyteorg/flyte --kubeconfig=~/.flyte/k3s/k3s.yaml
+
+   .. tabbed:: AWS/GCP
+
+        .. code-block:: bash
+
+            helm upgrade -n flyte -f values-override.yaml flyteorg/flyte-core --kubeconfig=~/.flyte/k3s/k3s.yaml
+
+
+
+6. Register the plugin example.
+
+   .. tabbed:: PyTorch Operator
+
+        .. code-block:: bash
+
+           flytectl register files --config ~/.flyte/flytectl.yaml https://github.com/flyteorg/flytesnacks/releases/download/v0.2.225/snacks-cookbook-integrations-kubernetes-kfpytorch.tar.gz --archive -p flytesnacks -d development --version latest
+
+   .. tabbed:: Tensorflow Operator
+
+        .. code-block:: bash
+
+           # TODO: https://github.com/flyteorg/flyte/issues/1757
+           flytectl register files --config ~/.flyte/flytectl.yaml https://github.com/flyteorg/flytesnacks/releases/download/v0.2.225/<TODO>.tar.gz --archive -p flytesnacks -d development --version latest
+
+   .. tabbed:: MPI Operator
+
+        .. code-block:: bash
+
+           flytectl register files --config ~/.flyte/flytectl.yaml https://github.com/flyteorg/flytesnacks/releases/download/v0.2.226/snacks-cookbook-integrations-kubernetes-kfmpi.tar.gz --archive -p flytesnacks -d development --version latest
+
+   .. tabbed:: Spark Operator
+
+        .. code-block:: bash
+
+           flytectl register files --config ~/.flyte/flytectl.yaml https://github.com/flyteorg/flytesnacks/releases/download/v0.2.226/snacks-cookbook-integrations-kubernetes-k8s_spark.tar.gz --archive -p flytesnacks -d development --version latest
+
+
+7. Launch an execution
+
+   .. tabbed:: Flyte Console
```

Comment:
Can we separate Flyte Console and FlyteCTL? For example, we can have two bullet points, one for Flyte Console and the other for FlyteCTL. Within FlyteCTL, we can have tabs.

Reply:
yes